drivers/power_button.c

```diff
@@ -0,0 +1,60 @@
+#include <acpi_ktf.h>
+#include <drivers/power_button.h>
+#include <spinlock.h>
+
+static void default_handler(void *);
+
+spinlock_t pb_handler_lock = SPINLOCK_INIT;
```

Comment:
`pb_handler_lock` isn't used any more, can remove its definition as well.

Reply:
oups, removed it